### PR TITLE
Implement workflow for setting recipe version

### PR DESCRIPTION
**Pull request by dudoslav**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    if: ${{ github.event.client_payload.passed }}
+    runs-on: ubuntu-latest
+    env:
+      VERSION: ${{ github.event.client_payload.version }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Set version
+        run: |
+          sed -i "s/version = \".*\"/version = \"$VERSION\"/g" recipe/meta.yaml
```
Comment (jdblischak) [inline .github/workflows/release.yml:15]: You'll also need to ensure that the `build_number` is reset to `0` when auto-bumping the version

**Review by jdblischak — CHANGES_REQUESTED** [review]

**Comment by jdblischak** [comment]
In case it could be useful, here is some code I use to edit conda recipes.

The quickest option is to use `sed`. I like this approach for its simplicity, but it's also caused me problems when the file has been edited, and the patterns no longer match. Here's the [code](https://github.com/TileDB-Inc/conda-forge-nightly-controller/blob/51519a0f8340b32cf737fcb59b76c6a91c42dc47/scripts/tiledb/update-recipe.sh#L21) for bumping the build number:

```sh
# Ensure build number is 0
sed -i \
  s/"  number: [0-9]\+"/"  number: 0"/ \
  recipe/meta.yaml
```

A more robust option is to parse the recipe with ruamel.yaml, but this requires Python.  The payoff is that the [code](https://github.com/TileDB-Inc/conda-forge-nightly-controller/blob/main/scripts/tiledb-py/update-recipe.py) is much more robust and easier to read.

**Comment by dudoslav** [comment]
I am currently investigating if we could rework https://github.com/dudoslav/cf-scripts in order to create PRs for our repos so this PR is inactive for few days.

**Comment by dudoslav** [comment]
Closing this PR since this should be implemented by conda forge autobot

**Pull request closed without merging**